core/common/src/ConcurrentQuery.ts

```diff
@@ -45,12 +45,19 @@ export interface QueryLimit {
 
 /** @public */
 export interface QueryPropertyMetaData {
+  /** The class name is set to empty if the property is a generated one, otherwise, it is the name of the ECClass that the property is contained within. */
   className: string;
+  /** Access string is the property's display label if the property is a generated one, otherwise it is the ECSQL property path. */
+  accessString?: string;
+  /** Indicates whether the property is a generated one or whether it directly refers to an existing ECProperty of one of the classes in the FROM or JOIN clauses. */
   generated: boolean;
   index: number;
```

Comment:
This should be the index of the property value if the result is formatted as an array

Reply:
For an index more appropriate description would be - the ECSQL column index in the result set of the query.

Reply:
Updated the document as per suggestion